package/pyproject.toml

```diff
@@ -4,13 +4,10 @@ requires = [
     "Cython>=0.28",
     "packaging",
     # lowest NumPy we can use for a given Python,
-    # except for more exotic platforms (linux/Mac ARM flavors)
-    "numpy==1.19; python_version=='3.8' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64'",
```

Comment:
@tylerjereddy I wasn't 100% sure of the logic here, we seem to have entries specfic for "not ((arm64 or Darwin) and aarch64)" and "arm64 and Darwin" but nothing for aarch64. Is there a specific reason we were restricting aarch64 to py3.10 only?